Sources/Widgets/Widgets3D/LabelWidget/behavior.js

```diff
@@ -2,8 +2,7 @@ import macro from 'vtk.js/Sources/macros';
 
 export default function widgetBehavior(publicAPI, model) {
   model.classHierarchy.push('vtkLabelWidgetProp');
-
-  model.isDragging = null;
+  let isDragging = null;
```

Comment:
are you sure there is no derived widget that could need to access `isDragging` ?

Reply:
Indeed, derived widgets may need to access `isDragging` while overriding some methods. 
I have moved it to the model as a protected attribute (`model._isDragging`).